src/ray/core_worker/transport/raylet_transport.cc

```diff
@@ -37,14 +37,14 @@ CoreWorkerRayletTaskReceiver::CoreWorkerRayletTaskReceiver(
 
 void CoreWorkerRayletTaskReceiver::HandleAssignTask(
     const rpc::AssignTaskRequest &request, rpc::AssignTaskReply *reply,
-    rpc::RequestDoneCallback done_callback) {
+    rpc::SendReplyCallback send_reply_callback) {
   const std::string &task_message = request.task_spec();
   const raylet::Task task(*flatbuffers::GetRoot<protocol::Task>(
       reinterpret_cast<const uint8_t *>(task_message.data())));
   const auto &spec = task.GetTaskSpecification();
 
   auto status = task_handler_(spec);
-  done_callback(status);
+  send_reply_callback(status, nullptr, nullptr);
```

Comment:
can we use `nullptr` as default value?

Reply:
It seems that default value is not allowed in lambda function in c++11

Reply:
It's allowed.
```
    auto f = [](int a = 1) {
        std::cout << a << std::endl;
    };
    f();
```
prints `1`.